### ISSUE TITLE
Job Overview Table malfunctioning on iOS

### ISSUE THREAD

**Issue by nixnuex** [juntagrico/juntagrico#480]
In the job overview, on smartphones & tablets (e.g. iOS), the job description is visible in the table on the left exploding the row size beyond usable. On Desktop (Safari, Firefox) no description is visible. Seems to have been introduced with 1.4.6 through "Jobs table is now responsive".

What is the intended behavior?

**Comment by nixnuex** [comment]
Might be a cache issue. Seems
To be solved after emptying. But probably many users run into this.

**Comment by pascalfree** [comment]
Yes, it's a caching issue. Not sure if there is much we can do except for waiting until the cache expires

**Comment by nixnuex** [comment]
We should link static files with e.g. a version string, so that browsers load the fresh version on new releases:

company.com/static/css/index.css?version=1.0.0

**Comment by nixnuex** [comment]
On the activity area views, the job description is not collapsing after clearing the cache: under /my/area/*

**Comment by pascalfree** [comment]
On the activity area view there is a bug: The script path to the js file is wrong. I fixed that in #482

Also I made changes that allow using the setting
`STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'`
Setting it fixes the caching issue: https://docs.djangoproject.com/en/3.1/ref/contrib/staticfiles/#manifeststaticfilesstorage

**Comment by motu55** [comment]
this seems like a good solution, but only if the admins set this setting.
so the question remains how do we get the admins to set this setting
in my ipinion we have to adapt the following sections or parts:
the first steps section
the cookie cutter

also since we probably do not change the js in 1.4.x anymore (or if we do we move them to 1.5) would it make sense to include this to 1.5 and include the setting in the upgrade section so that all the admins will set it?

**Comment by pascalfree** [comment]
Yes we should add that to the docs and refer to it in the release notes and inform the admins specifically on that.

When the admins ignore it, they will run into the issue, ask for help and we can just point to the docs.

I agree with updating the cookie cutter. 

We can't wait for 1.5 as we already have this issue in 1.4.6, unless we fix it differently here.
